test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultTemplateEngineFactoryServiceTest.cs

```diff
@@ -128,38 +107,6 @@ public void Create_DoesNotSupportViewComponentTagHelpers_ForVersion1_0()
             Assert.Empty(engine.Engine.Features.OfType<Mvc1_X.ViewComponentTagHelperPass>());
         }
 
-        [Fact]
-        public void Create_HigherMvcVersion_UsesLatest()
```

Comment:
This scenario doesn't need to be handled by this class anymore. The project system deals with unsupported versions now.